service/lib/ops/teamOps.js

```diff
@@ -336,71 +398,61 @@ var teamOps = {
   },
 
   /**
-   * Fetch specific team data
+   * Fetch the users data from a team
    *
-   * @param  {params}   params { id, organizationId }
+   * @param  {params}   params { id, page, limit }
    * @param  {Function} cb
    */
-  readTeam: function readTeam ({ id, organizationId }, cb) {
-    const tasks = []
-    let team
-
-    tasks.push((next) => {
-      const sql = SQL`
-        SELECT *
-        FROM teams
-        WHERE id = ${id}
-        AND org_id = ${organizationId}
-      `
-
-      db.query(sql, (err, result) => {
-        if (err) return next(Boom.badImplementation(err))
-        if (result.rowCount === 0) return next(Boom.notFound(`Team with id ${id} could not be found`))
-
-        team = mapping.team(result.rows[0])
+  readTeamUsers: function readTeamUsers ({ id, page = 1, limit = 0 }, cb) {
+    const resultLimit = limit < 0 ? 0 : limit
+    const pageNumber = page < 1 ? 1 : page
+    const offset = pageNumber * limit - limit
+
+    const job = {
+      id: id,
+      offset: offset,
+      limit: resultLimit,
+      team: {}
+    }
 
-        team.users = []
-        team.policies = []
-        next()
-      })
+    loadTeamUsers(job, (err) => {
+      if (err) return cb(err)
+      const pageSize = resultLimit || job.totalUsersCount
+      const result = {
+        currentPage: pageNumber,
+        pageSize: pageSize,
+        totalPages: Math.ceil(job.totalUsersCount / pageSize),
+        totalUsersCount: job.totalUsersCount,
+        users: job.team.users.map(user => Object.assign({}, user))
```

Comment:
@paolochiodi seemed to me that this is the first place where we build the content of an object for returning. In the other places the content is mostly built for us (based on mappings). On the same idea (creating new data) felt that I should deep clone it not just reference it but no other reasons behind it.